pytests/test_wff_parental_tax_credit.py

```diff
@@ -22,13 +22,161 @@ class TestKey(Reasoner):
 """
 
 
+class TestWFF_ParentalTaxCreditDefault(TestKey):
+    body = {}
+
+    def test_reasoning(self):
+        self.assertTrue(self.is_forbidden)
+
+
 class TestWFF_ParentalTaxCredit(TestKey):
     body = {
         "applicant": {
             "isPrincipalCarerForProportion": 34,
             "receivesIncomeTestedBenefit": False
+        },
+        "threshold": {
+            "income": {
+                "workingForFamiliesParentalTaxCredit": True
+            }
         }
     }
 
     def test_reasoning(self):
         self.assertTrue(self.is_permitted)
+
+
+class TestWFF_ParentalTaxCreditNullThreshold(TestKey):
+    body = {
+        "applicant": {
+            "isPrincipalCarerForProportion": 34,
+            "receivesIncomeTestedBenefit": False
+        }
+    }
+
+    def test_reasoning(self):
+        self.assertTrue(self.is_forbidden)
+
+
+class TestWFF_ParentalTaxCreditNotThreshold(TestKey):
+    body = {
+        "applicant": {
+            "isPrincipalCarerForProportion": 34,
+            "receivesIncomeTestedBenefit": False
+        },
+        "threshold": {
+            "income": {
+                "workingForFamiliesParentalTaxCredit": False
+            }
+        }
+    }
+
+    def test_reasoning(self):
+        self.assertTrue(self.is_forbidden)
+
+
+class TestWFF_ParentalLessChildcare(TestKey):
+    body = {
+        "applicant": {
+            "isPrincipalCarerForProportion": 4,
+            "receivesIncomeTestedBenefit": False
+        },
+        "threshold": {
+            "income": {
+                "workingForFamiliesParentalTaxCredit": True
+            }
+        }
+    }
+
+    def test_reasoning(self):
+        self.assertTrue(self.is_forbidden)
+
+
+class TestWFF_ParentalTaxCreditOnMTBenefit(TestKey):
+    body = {
+        "applicant": {
+            "isPrincipalCarerForProportion": 100,
+            "receivesIncomeTestedBenefit": True
+        },
+        "threshold": {
+            "income": {
+                "workingForFamiliesParentalTaxCredit": True
+            }
+        }
+    }
+
+    def test_reasoning(self):
+        self.assertTrue(self.is_forbidden)
+
+
+class TestWFF_ParentalTaxCreditForbiddenOnAcc(TestKey):
+    body = {
+        "applicant": {
+            "isPrincipalCarerForProportion": 34,
+            "receivesIncomeTestedBenefit": False,
+            "isOnACCCompensation": True
+        },
+        "threshold": {
+            "income": {
+                "workingForFamiliesParentalTaxCredit": True
+            }
+        }
+    }
+
+    def test_reasoning(self):
+        self.assertTrue(self.is_forbidden)
+
+
+class TestWFF_ParentalTaxCreditForbiddenOrphans(TestKey):
+    body = {
+        "applicant": {
+            "isPrincipalCarerForProportion": 34,
+            "receivesIncomeTestedBenefit": False,
+            # add the orphans bene requirements
+            "Age": 87,
+            "isNZResident": True,
+            "isParent": False,
```

Comment:
I had a mistake at my end where I was only sending this if it was true (as it piggybacks on another atom), I'm fixing it today.